File: src/jl/pe_0002.jl
function main()
    gross = 0
    a = 1
    b = 2

    while a < 4e6
        if a % 2 == 0
            gross += a
        end
        c = a + b
        a = b
        b = c
    end

    println(gross)
end

main()

File: src/c/pe_0002.c
#include <stdio.h>


// SCRIPT


int main() {
    int gross, a, b, c;

    gross = 0;
    a = 1;
    b = 2;
    while (a < 4e6) {
        if (a % 2 == 0) {
            gross += a;
        }
        c = a + b;
        a = b;
        b = c;
    }

    printf("%d\n", gross);
}
